deps/mbedtls.mk

```diff
@@ -56,8 +52,10 @@ endif
 	echo 1 > $@
 
 $(MBEDTLS_OBJ_TARGET): $(MBEDTLS_OBJ_SOURCE) | $(build_shlibdir)
-ifeq ($(OS), WINNT)
+ifeq ($(BUILD_OS), WINNT)
```

Comment:
I'll put it back for now though since it's putting .dll.a files in the wrong place otherwise, need to fix that upstream. I plan on moving appveyor over to a cygwin-cross and dropping official support for MSYS2 in the future since it's not actively maintained and is the cause of too many of these special cases.